examples/cloudaccounts/k8s/README.md

```diff
@@ -0,0 +1,67 @@
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/dome9/dome9-sdk-go/dome9"
+	"github.com/dome9/dome9-sdk-go/services/cloudaccounts"
+	"github.com/dome9/dome9-sdk-go/services/cloudaccounts/k8s"
+)
+
+func main() {
+	// Pass accessID, secretKey, rawUrl, or set environment variables
+	config, _ := dome9.NewConfig("", "", "")
+	srv := k8s.New(config)
+	var createAccountReqBody k8s.CloudAccountRequest
+	
+	createAccountReqBody.Name = "ACCOUNT NAME" //mandatory field
```

Comment:
call it cluster name instead of account name.
in dome9 it is called cloudAccount but we try as we can to call it to customer as cluster name